randovania/interface_common/options.py

```diff
@@ -131,6 +131,8 @@ def from_json(cls, value: dict) -> PerGameOptions:
     "dark_mode": Serializer(identity, bool),
     "show_multiworld_banner": Serializer(identity, bool),
     "experimental_settings": Serializer(identity, bool),
+    "sound_alerts": Serializer(identity, bool),
+    "visual_taskbar_dock_alerts": Serializer(identity, bool),
```

Comment:
I would prefer to call these everywhere as `sound_generation_alert` (maybe audio instead?) and `visual_generation_alert`

Reply:
> sound_generation_alert (maybe audio instead?)  

Gonna go for "audible_generation_alert" then